crash-monitor.sh

```
@@ -0,0 +1,338 @@
+    echo -e "${YELLOW}📈 Project Complexity Analysis${NC}"
+    
+    # Count files by type
+    echo -e "${BLUE}File counts:${NC}"
+    echo "TypeScript files: $(find . -name "*.ts" -not -path "./node_modules/*" | wc -l)"
+    echo "TypeScript React files: $(find . -name "*.tsx" -not -path "./node_modules/*" | wc -l)"
+    echo "JavaScript files: $(find . -name "*.js" -not -path "./node_modules/*" | wc -l)"
+    echo "JavaScript React files: $(find . -name "*.jsx" -not -path "./node_modules/*" | wc -l)"
+    
+    # Find largest files
+    echo -e "${BLUE}Largest TypeScript files:${NC}"
+    find . -name "*.ts" -o -name "*.tsx" -not -path "./node_modules/*" | xargs ls -la 2>/dev/null | sort -k5 -nr | head -10 | awk '{print $5, $9}'
```

Shellcheck (0.10.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)